documentstore_migracao/processing/inserting.py

```diff
@@ -0,0 +1,64 @@
+""" module to processing to inserting methods """
+
+import os
+import logging
+
+from documentstore_migracao.utils import files, xml
+from documentstore_migracao import config
+
+
+logger = logging.getLogger(__name__)
+
+
+def document_store_by_database(folder: str, session_db, storage) -> None:
+
+    logger.info("Processando a Pasta %s", folder)
+    list_files = files.list_files(folder)
+
+    obj_xml = None
+    prefix = ""
+    xml_files = files.xml_files_list(folder)
+    medias_files = [mf for mf in list_files if mf not in xml_files]
+
+    for x_file in xml_files:
+        xml_path = os.path.join(folder, x_file)
+
+        obj_xml = xml.loadToXML(xml_path)
+        scielo_id = xml.get_scielo_id(obj_xml)
+        acrom = xml.get_journal_id(obj_xml)
+        issn = xml.get_issn_journal(obj_xml)
+
+        prefix = f"{issn}_{acrom}/{scielo_id}"
+        url_xml = storage.register(xml_path, prefix)
+
```

Comment:
`url_xml` dentro do `for`, contém somente o último valor

Reply:
@robertatakenaka fiz uns ajustes